app/Http/Controllers/System/AssessmentController.php

```diff
@@ -14,7 +14,7 @@ class AssessmentController extends Controller
     public function index()
     {
         $assesment = assessment::all();
```

Comment:
Fix all variable names in controller, make sure spelling is correct.

Reply:
Also delete the storage folder in the root directory before pushing to resolve merging conflicts.

Reply:
Fixed the above changes